mautic/mautic
# Query optimization campaign stat summary

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [N]
| New feature/enhancement? (use the a.x branch)      | [Y]
| Deprecations?                          | [N]
| BC breaks? (use the c.x branch)        | [N]
| Automated tests included?              | [N] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


This will improve the query performance of campaign stat summary query

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Run campaign to add records to campaign_lead_event_log table.
3. Run the 'mautic:campaigns:summarize' command and see the query performance




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11282"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
